### ISSUE TITLE
Trouble in Training in Getting Started

### ISSUE THREAD

**Issue by moordo91** [exiawsh/StreamPETR#117]
Hello. I apologize for the annoyance, but I am having difficulty running StreamPETR. I was following the 'getting started' you provided and was successful in training the model, but then I ran into an issue where no training is happening at all. Specifically, the loss is not reduced at all.

I am attaching [my log file](https://github.com/moordo91/StreamPETR/blob/main/logs/20231107_235103.log).

I downloaded all the pkl files and pth files from your GitHub link.

I ran the 'Train' process using the following commands.

```sh
CUDA_VISIBLE_DEVICES=3 PORT=30500 tools/dist_train.sh projects/configs/StreamPETR/stream_petr_r50_flash_704_bs2_seq_24e.py 1 --work-dir work_dirs/stream_petr_r50_flash_704_bs2_seq_24e/
```

I was wondering if you could provide any ideas or hints?

If you have any further questions for me, I will do my best to answer them.

**Comment by exiawsh** [comment]
@moordo91 Hi, thanks for your attention. Try to adjust the hyper-parameter. Because you only have 1 40G A100 for training.
you can set samples_per_gpu=8,
optimizer = dict(
    type='AdamW',
    lr=2e-4,
    paramwise_cfg=dict(custom_keys=dict(img_backbone=dict(lr_mult=0.25))),
    weight_decay=0.01)

**Comment by moordo91** [comment]
Today morning, I seem to have gotten [some good results](https://github.com/moordo91/StreamPETR/blob/main/logs/stream_petr_r50_flash_704_bs2_seq_24e/20231108_213859.log). 
Thank you so much for your kind response! 
I would have had a hard time figuring it out without your response.😄😄😄

**Comment by exiawsh** [comment]
@moordo91 Feel free to contact us if you have any problems. You provided enough details, so I can identify the problem.
